docs/gettingstarted.rst

```diff
@@ -0,0 +1,91 @@
+Getting started with ilastik API and OMERO
+==========================================
+
+**Description**
+---------------
+
+We will use a Python script showing how to analyze data stored in an OMERO server
+using the ilastik API. The code snippets are extracted from the Python script
+:download:`pixels_classification.py <../scripts/pixels_classification.py>`. A notebook
+is also available, see `pixels_classification.ipynb <https://mybinder.org/v2/gh/ome/omero-guide-ilastik/master?filepath=notebooks>`_.
+
+
+We will show:
+
+- How to connect to server
+
+- How load images from a dataset using the OMERO API.
+
+- How to run ilastik using its Python API.
+
+- How to save the generated results as OMERO images.
+
+**Resources**
+-------------
+
+We will use an ilastik project created with ilastik version 1.3.3 to analyse 3D tissues
```

Comment:
Confusingly, this is not considered to be a ``tissue`` and it is not in Tissue-IDR (rather Cell-IDR). Maybe

``...analyse 3D tissues data from the Image Data Resource (IDR).`` ->  ``...analyze 3D images of mouse blastocysts from the Image Data Resource (IDR)``